.cargo/xtask.toml

```diff
@@ -13,6 +13,7 @@
 [libraries."libgcc_s.so.1"]
 [libraries."libipcc.so.1"]
 [libraries."libkstat.so.1"]
+[libraries."liblzma.so.5"]
```

Comment:
@papertigers can I just... add this? @rcgoodfellow this is due to a new dep from falcon from https://github.com/oxidecomputer/falcon/pull/88

Reply:
That's an explicit allow list, so as long as we expect that lib to be present in the gz and ngz it's the right place to stick it. If it's a lib we don't want to leak out of a single binary we can add it an allow list similar to what libipcc has.